python/cudnn/gemm/cutedsl/grouped/dswiglu/api.py

```
@@ -679,8 +690,8 @@ def grouped_gemm_dswiglu_wrapper_sm100(
     beta_tensor: Optional[torch.Tensor],
     prob_tensor: torch.Tensor,
     norm_const_tensor: Optional[torch.Tensor] = None,
-    acc_dtype: torch.dtype = torch.float32,
-    d_dtype: torch.dtype = torch.bfloat16,
+    acc_dtype: Optional[torch.dtype] = None,
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)